t/api.t

```diff
@@ -68,6 +68,25 @@ subtest 'declared collections' => \&test_api,
     } ),
     '/yancy/api';
 
+if ($ENV{REGEN_OPENAPI}) {
```

Comment:
Could we make this a script just like the `t/schema/dbicdump.pl`? Something like `t/schema/openapi.pl`? I'm not so keen on setting an envvar to get a side-effect from the test.

Reply:
I will do this.